debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnectorConfig.java

```diff
@@ -410,6 +410,14 @@ public static SecureConnectionMode parse(String value, String defaultValue) {
                                                            .withDependents(DATABASE_WHITELIST_NAME)
                                                            .withDescription("Flag specifying whether built-in tables should be ignored.");
 
+    public static final Field JDBC_DRIVER = Field.create("database.jdbc.driver")
+            .withDisplayName("Jdbc Driver Class Name")
+            .withType(Type.STRING)
```

Comment:
Also, I think it'd be better to use `Type.CLASS` here rather than `Type.STRING`, since the former basically specifies a literal string that is a class name.